commands/vpc_peerings.go

```diff
@@ -0,0 +1,203 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/doctl/do"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+)
+
+// VPCPeerings creates the vpc peerings command.
+func VPCPeerings() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:   "vpc-peerings",
+			Short: "Display commands that manage VPC Peerings",
+			Long: `The commands under ` + "`" + `doctl vpc peerings` + "`" + ` are for managing your VPC Peerings.
+
+With the VPC Peerings commands, you can get, list, create, update, or delete VPC Peerings, and manage their configuration details.`,
+			GroupID: manageResourcesGroup,
+		},
+	}
+
+	peeringDetails := `
+- The VPC Peering ID
+- The VPC Peering Name
+- The First VPC network's ID
+- The Second VPC network's ID
+- The VPC Peering Status
+- The VPC Peering creation date, in ISO8601 combined date and time format
+`
+	cmdPeeringGet := CmdBuilder(cmd, RunVPCPeeringGet, "get <id>",
+		"Retrieve a VPC network", "Retrieve information about a VPC Peering, including:"+peeringDetails, Writer,
+		aliasOpt("g"), displayerType(&displayers.VPCPeering{}))
+	cmdPeeringGet.Example = `The following example retrieves information about a VPC Peering with the ID ` + "`" + `f81d4fae-7dec-11d0-a765-00a0c91e6bf6` + "`" + `: doctl vpc-peerings get f81d4fae-7dec-11d0-a765-00a0c91e6bf6`
+
+	cmdPeeringList := CmdBuilder(cmd, RunVPCPeeringList, "list", "List VPC Peerings", "Retrieves a list of the VPC Peerings on your account, including the following information for each:"+peeringDetails, Writer,
+		aliasOpt("ls"), displayerType(&displayers.VPC{}))
+	AddStringFlag(cmdPeeringList, doctl.ArgVPCPeeringVPCID, "", "",
+		"VPC ID")
+	cmdPeeringList.Example = `The following example lists the VPC Peerings on your account : doctl vpc-peerings list --format Name,FirstVpcID,SecondVpcID`
+
+	cmdPeeringCreate := CmdBuilder(cmd, RunVPCPeeringCreate, "create",
+		"Create a new VPC Peering", "Use this command to create a new VPC Peering on your account.", Writer, aliasOpt("c"))
+	AddStringFlag(cmdPeeringCreate, doctl.ArgVPCPeeringName, "", "",
+		"The VPC Peering's name", requiredOpt())
+	AddStringFlag(cmdPeeringCreate, doctl.ArgVPCPeeringFirstVPCID, "", "",
+		"First VPC ID")
+	AddStringFlag(cmdPeeringCreate, doctl.ArgVPCPeeringSecondVPCID, "", "",
+		"Second VPC ID")
+	cmdPeeringCreate.Example = `The following example creates a VPC Peering named ` +
+		"`" + `example-peering` + "`" +
+		` : doctl vpc-peerings create --name example-peering --first-vpc-id f81d4fae-7dec-11d0-a765-00a0c91e6bf6 --second-vpc-id 3f900b61-30d7-40d8-9711-8c5d6264b268`
+
+	cmdPeeringUpdate := CmdBuilder(cmd, RunVPCPeeringUpdate, "update <id>",
+		"Update a VPC Peering's configuration", `Updates a VPC network's configuration. You can update its name.`, Writer, aliasOpt("u"))
```

Comment:
wondering if we should rephrase this line,  as we really cannot `Updates a VPC network's configuration` through this